docs/broker/README.md

```diff
@@ -0,0 +1,31 @@
+# Knative Eventing Broker for Apache Kafka
+
+This document gives a rough overview over the architecture of the Knative Eventing Broker for Apache Kafka.
+
+![Overview](eventing-kafka-broker.png)
+
+## Control Plane
+
+The `kafka-controller` is kind of the heart of the Broker for Apache Kafkas control plane and has the following responsibilities:
+
+* Watch the following resources and update the `kafka-broker-brokers-triggers` configmap (step 3 in the diagram) with relevant information about the resource:
+  * `Broker`s with the `eventing.kantive.dev/broker.class: Kafka` annotation (step 2 in the diagram),
+  * `Trigger`s referencing a Knative Eventing Broker for Apache Kafka,
+  * `KafkaChannel`s,
+  * `KafkaSink`s 
+  * `KafkaSource`s
+* Update the status on `Broker` resources with the `eventing.kantive.dev/broker.class: Kafka` annotation with the address for the broker ingress.
```

Comment:
I think I'd also list that `kafka-broker-config` there, and mention that if not provided we use the one on the SYSTEM NS.

I am a huge fan of having a 1:1 coupling of CM-Broker

Reply:
I'd also argue not not use `default` NS, but a different one, to make things a bit more explicit?

Reply:
> I'd also argue not not use default NS, but a different one, to make things a bit more explicit?

:+1: updated